Formula/ldc.rb

```diff
@@ -5,8 +5,10 @@ class Ldc < Formula
   stable do
     # for the sake of LLVM 3.9 compatibility
     url "https://github.com/ldc-developers/ldc.git",
-        :branch => "release-1.1.0"
-    version "1.1.0"
+        :branch => "release-1.1.0",
+        :revision => "a86f87637d4880709ee0bde3ae741e6617b835b2"
+    # NOTE sure what suffix should be after 1.1.0
+    version "1.1.0-alpha1"
```

Comment:
there's a catch 22 situation:
* initially I had made this as a linuxbrew only change https://github.com/Linuxbrew/homebrew-core/pull/1525 because it fixes a long standing issue (brew install ldc has been broken for months https://github.com/Linuxbrew/homebrew-core/issues/852)
* but @maxim-belkin  told me to make this a homebrew PR which I did
* if this remains as devel version, linuxbrew ldc remains broken
* furthermore, before this PR, the code was:

:branch => "release-1.0.1",
```
:branch => "release-1.0.1",
    :revision => "3461e00f3531f855f9fc6e92515d7affb8201827"
    version "1.0.1-alpha1"
```

so the `stable` branch was already not a stable release. 
This PR doesn't make anything worse, and it fixes linuxbrew ldc which was broken for months.

Please advise